core/ir/aarch64/codec.c

```diff
@@ -3652,6 +3671,52 @@ encode_opnd_vindex_H(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *enc_
     return true;
 }
 
+/* svemem_gpr_simm: 9 bit signed immediate offset added to base register defined in bits 5
+ * to 9.
+ */
+#if !defined(DR_HOST_NOT_TARGET) && !defined(STANDALONE_DECODER)
+/* i3044 TODO: Vector length will be read from cpuinfo, e.g.
+ * opnd_size_from_bytes(proc_get_vector_length()));
+ * Setting to fixed size for now in order to pass unit tests.
+ */
+#    define OPSZ_SVE opnd_size_from_bytes(32)
+#else
+/* i3044 TODO: How do we set SVE vector length for offline decode?
+ * A command line option?
+ */
+#    define OPSZ_SVE opnd_size_from_bytes(32)
+#endif
+
+static inline bool
+decode_opnd_svemem_gpr_simm(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
```

Comment:
There are gpr_simm signatures without a mul vl operand, so this should reflect it is mul vl in the name

Reply:
Done.